app/services/transformers.py

```diff
@@ -1233,47 +1329,51 @@ def __init__(self, *, config: SMARTPushEventActionConfig, **kwargs):
 
     async def transform(
         self, message: schemas.v2.Event, rules: list = None, **kwargs
-    ) -> dict:
+    ) -> SMARTCompositeRequest:
+        if self._version and version.parse(self._version) < version.parse("7.5"):
+            raise ValueError("Smart version < 7.5 is not supported")
```

Comment:
@marianobrc FYI, these version checks are vestiges we can eliminate whenever it suits us. _We now only support >=7.5 and I think it's safe to assume all the Smart Connect servers in use are beyond that._

Reply:
Thanks for letting me know that @chrisdoehring. If you think it's safe I can remove them, but since some servers are manually installed, can we be sure every Smart Connect server in the world was updated? Maybe we can leave the validation for a little bit longer and check if we catch any of those in the next months before removing it?